libs/sdk-core/src/models.rs

```diff
@@ -542,12 +542,32 @@ impl From<Network> for bitcoin::network::constants::Network {
     }
 }
 
+#[derive(PartialEq)]
+pub enum JsonFieldValue {
+    IsNull,
+    Bool { value: bool },
+    Number { value: u64 },
+    String { value: String },
+}
```

Comment:
Just wanted to leave two quick notes regarding the enum:
1. The choice to have the values specified in such a way (using a struct with `value` field) is due to UniFFI's limitations in regards to enums.
2. When `IsNull` is specified, due to limitations in SQLite's json methods when a field does not exist in an object, it will be treated as if it were null (e.g. in the following example, querying for "field" IsNull will return both objects): 
```js
  obj1 = {
     "field": null  
  }
  obj2 = {}
  ```
This is not a problem in most scenarios, especially when all the payment metadatas retain the same structure, but I believe it should be properly documented anyway.